src/auto-evo/AutoEvoConfiguration.cs

```diff
@@ -1,48 +1,49 @@
-﻿using Newtonsoft.Json;
+﻿using System;
+using Newtonsoft.Json;
 
-public class AutoEvoConfiguration : IRegistryType
+public class AutoEvoConfiguration : IRegistryType, ICloneable
 {
     [JsonProperty]
-    public int MutationsPerSpecies { get; private set; }
+    public int MutationsPerSpecies { get; set; }
```

Comment:
Now that everything is publicly settable here, could you add a comment on this class like:
```
/// <summary>
///   Settings for auto-evo
/// </summary>
/// <remarks>
///   <para>
///      Like all registry types this should not be modified during runtime at all. The reason the properties are modifiable is to allow auto-evo exploring tool to modify its own copy of the settings
...
```